server/methods.js

```diff
@@ -759,5 +759,35 @@ Meteor.methods({
     }
     console.log("OK");
     return true;
+  },
+  instanceAdminCheck(instanceid, adminId) {
```

Comment:
@singhsanket143 you don't need to (and shouldn't) pass the `adminId` here. Instead, us `Meteor.userId()` to get the current user id.